### ISSUE TITLE
headless and build commands are inverted

### ISSUE THREAD

**Issue by alexandrpunk** [vituum/vituum#9]
I was trying to build some templates with the command vituum build , but every time, the views folder from the output directory was been deleted. It was until I tried the command vituum headless that finally the views folder wasn't deleted.

**Comment by lubomirblazekcz** [comment]
it's not deleted, the files from views should be copied to output directory directly without the views dir. And headless build should not build views dir at all, it's a headless build without templates.

**Comment by isspe-escolar** [comment]
> it's not deleted, the files from views should be copied to output directory directly without the views dir. And headless build should not build views dir at all, it's a headless build without templates.
Yes, that's the behavior I was expecting after reading the documentation, but as I explained the commands are inverted, this monday I will try to upload a video to show the problem.

**Comment by alexandrpunk** [comment]
> it's not deleted, the files from views should be copied to output directory directly without the views dir. And headless build should not build views dir at all, it's a headless build without templates.

Yes, that's the behavior I was expecting after reading the documentation, but as I explained the commands are inverted, this monday I will try to upload a video to show the problem.

**Comment by lubomirblazekcz** [comment]
this might be related to https://github.com/vituum/vituum/issues/4, there seems to be problem on Windows

**Comment by alexandrpunk** [comment]
> this might be related to #4, there seems to be problem on Windows

Yes, the issue was in windows

**Comment by lubomirblazekcz** [comment]
should be fixed now in https://github.com/vituum/vituum/releases/tag/0.0.38
